src/qnetvo/cost/negativity.py

```diff
@@ -0,0 +1,69 @@
+import pennylane as qml
+from pennylane import math
+from pennylane import numpy as np
+from ..qnodes import reduced_density_matrix_qnode
+
+def negativity_cost_fn(network_ansatz, m, n, wires, qnode_kwargs={}):
+    """Constructs an ansatz-specific negativity cost function.
+
+    Negativity can be used to identify if two subsystems :math:`A` and :math:`B` are entangled, through the PPT criterion. Negativity is an upper bound for distillable entanglement.
```

Comment:
If there is an arxiv paper for this or other good online resource, please add a link